doodle.py

```diff
@@ -65,13 +65,51 @@ def main():
 
 # Parse date string and return date and whether it was relative or not
 def derelativize_date(datestr: str, base=datetime.now()):
-    if datestr == '':
-        return base.replace(hour=0, minute=0, second=0, microsecond=0), True
+    relative = False
```

Comment:
I'd stip this and return the literal, since the variable is just set manually twice.